plezanje-net/web
# Admin overview WIP

- [ ] add link to overview for admins / users
- [ ] create contributions overview page with action links for status changes
- [x] create warnings on crags / sectors / routes that are private or in review



To test:
All tests should be performed by 4  different 'users':
- guest (not logged in)
- user (not an admin) that made some contributions
- another user (not an admin) that  made some other contributions
- admin

Homepage>
admins get noticed when there are some pending _in_review_ contributions
users get warned when they own some contributions in status _draft_

Route page>
users get warned when they own the route and is still in status draft
users get noticed when they own the route and is pending admin's review
admins get warned when they own the route and haven't published it yet
admins get noticed when they the route is pending review

Route list page>
users get warned when they own the route and is still in status draft
users get noticed when they own the route and is pending admin's review
admins get warned when they own the route and haven't published it yet
admins get noticed when they the route is pending review

Sector on route list page>
users get warned when they own the sector and is still in status draft
users get noticed when they own the sector and is pending admin's review
admins get warned when they own the sector and haven't published it yet
admins get noticed when they the sector is pending review

Crag page>
users get warned when they own the crag and is still in status draft
users get noticed when they own the crag and is pending admin's review
admins get warned when they own the crag and haven't published it yet
admins get noticed when they the crag is pending review

also
closes #377 
(missing ž will be manually set in db)
